packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -509,7 +515,7 @@ const Sidebar: React.FC<RouteComponentProps & SidebarProps> = props => {
                 }),
               }}
             >
-              {active ? 'Copied! Happy coding :)' : `${pkg.name}@${pkg.version}`}
+              {active ? 'Copied! Happy coding :)' : `${pkg.name}@${version}`}
```

Comment:
If we will hide `yarn add` (https://github.com/microsoft/fluentui/pull/16866/files#r572383896), we can restore this change

Reply:
Okay, will hide and restore :)